src/CommandingOfficers/DefendPeace/OS/Sami.java

```diff
@@ -0,0 +1,162 @@
+package CommandingOfficers.DefendPeace.OS;
+
+import java.util.ArrayList;
+
+import CommandingOfficers.*;
+import CommandingOfficers.AW1.OS.Sami.PerfectMoveModifier;
+import CommandingOfficers.AWBW.AWBWCommander;
+import Engine.GameScenario;
+import Engine.XYCoord;
+import Engine.Combat.StrikeParams;
+import Engine.Combat.StrikeParams.BattleParams;
+import Engine.GameEvents.GameEventQueue;
+import Engine.GameEvents.MassDamageEvent;
+import Engine.UnitMods.UnitDamageModifier;
+import Engine.UnitMods.UnitDefenseModifier;
+import Engine.UnitMods.UnitModifier;
+import Engine.UnitMods.UnitMovementModifier;
+import Engine.UnitMods.UnitTypeFilter;
+import Terrain.MapLocation;
+import Terrain.MapMaster;
+import Terrain.TerrainType;
+import UI.UIUtils;
+import Units.Unit;
+import Units.UnitContext;
+import Units.UnitModel;
+import lombok.var;
+
+public class Sami extends AWBWCommander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  public static CommanderInfo getInfo()
+  {
+    return coInfo;
+  }
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Sami", UIUtils.SourceGames.DEFEND_PEACE, UIUtils.OS, "VM");
+      infoPages.add(new InfoPage(
+            "Sami (victory)\n"
+          + "AW1 Sami, but with instant captures and AWBW rules.\n"
+          + "Footsoldiers +20/10 stats.\n"
+          + "20x capture rate, lose 5 HP on capture.\n"
+          + "Cities do not give income if there is any unit (friendly or not) on them.\n"
```

Comment:
How is this Sami themed?

Reply:
It is "make this design not T -1" themed.
It also makes her D2D somewhat symmetric, since she loses the city income as soon as you start capturing.

But yeah, it probably could be better.